util/guard-one/guard-one.repl

```diff
@@ -0,0 +1,172 @@
+;; gas-guards.repl
+(env-data
+ { 'util-ns-users: ["util-ns-user"]
+ , 'util-ns-admin: ["util-ns-admin"]
+ })
+
+(env-keys ["util-ns-user", "util-ns-admin"])
+
+;;Load contracts
+(begin-tx)
+(load "../util/util-ns.pact")
+(load "../guards/guards.pact")
```

Comment:
What is this for?

Reply:
Deleted it - guard1 had `(use guards)` in the module, but took it out because it was unneccesary